drivers/sftp/manifest.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "sftp",
+  "description": "Store files on a remote machine using SFTP.",
+  "options": {
+    "hostname": {
+      "name": "Hostname",
+      "description": "The hostname for the SFTP connection",
+      "type": "string"
+    },
+    "username": {
+      "name": "Username",
+      "description": "The username for the SFTP connection",
+      "type": "string"
+    },
+    "password": {
+      "name": "Password",
+      "description": "The password for the SFTP connection. You must use either this or a private key.",
+      "type": "string",
+      "default": ""
+    },
+    "port": {
+      "name": "Port number",
+      "description": "The port number for the SFTP connection",
+      "type": "integer",
+      "default": 22
+    },
+    "changes_timer": {
+      "name": "Timer for modification check",
+      "description": "Time in seconds between two checks for files modifications",
+      "type": "integer",
+      "default": 60
```

Comment:
I think 1 minute is not long enough. I don't think someone using SFTP is looking for a fast synchronization, so we can do it something like every 5 minutes or so.